openwisp_controller/config/static/config/js/widget.js

```diff
@@ -370,8 +370,10 @@
     };
 
     var bindLoadUi = function () {
+        var orgId = $('#id_organization').val();
         $('.jsoneditor-raw:not([name*="__prefix__"])').each(function (i, el) {
-            $.getJSON($(el).data('schema-url'), function (schemas) {
+            var url = $(el).data('schema-url') + '?organization_id=' + orgId;
+            $.getJSON(url, function (schemas) {
```

Comment:
@nemesisdesign we need to send the `organization_id` to the backend so we can fetch the commands available for the organization. I have implemented this in a brute force way right now. The `organization_id=<id>` keyword parameter is sent with both the requests: 

1. request to fetch configuration schema 
2. request to fetch command schema

Do you have any suggestions to improve this? 

I have following in mind: 

We add another attribute here in the field which will store the `id` of the HTML element whose value should be sent with the request. 

https://github.com/openwisp/openwisp-controller/blob/0e635978accf73291f91ccc13d8e80d3afc4fff6/openwisp_controller/config/widgets.py#L40-L54